docs/content/manual/_incoming/2750-support-audit-event-log.md

```diff
@@ -0,0 +1,257 @@
+                owner: 0
+                group: 0
+                content: |
+                apiVersion: audit.k8s.io/v1
+    ```
+    ```
+    vim /oem/99_custom.yaml
+    
+    - path: /etc/rancher/rke2/config.yaml.d/92-harvester-kube-audit-policy.yaml
+                permissions: 384
+                owner: 0
+                group: 0
+                content: |
+                apiVersion: audit.k8s.io/v1
+                kind: Policy
+                omitStages:
+                    - "ResponseStarted"
+                    - "ResponseComplete"
+                rules:
+                    # Any include/exclude rules are added here
+                    # A catch-all rule to log all other (create/delete/patch) requests at the Metadata level
+                    - level: Metadata
+                    verbs: ["create", "delete", "patch"]
+                    omitStages:
+                        - "ResponseStarted"
+                        - "ResponseComplete"
+                encoding: ""
+                ownerstring: ""
+    ```
+
+### Test Result 2: Multi nodes upgrade check 
+### Multi nodes upgrade check 
+1. Check the following files and pods have no error 
+1. logging related pods 
+    ```
+    n1-103:~ # kubectl get pods -n cattle-logging-system
+    NAME                                             READY   STATUS    RESTARTS      AGE
+    harvester-default-event-tailer-0                 1/1     Running   0             47m
+    rancher-logging-574448c578-wv9wd                 1/1     Running   0             47m
+    rancher-logging-kube-audit-fluentbit-dnxzr       1/1     Running   2 (38m ago)   59m
+    rancher-logging-kube-audit-fluentbit-s9bfj       1/1     Running   2 (29m ago)   59m
+    rancher-logging-kube-audit-fluentbit-snrml       1/1     Running   2 (49m ago)   59m
+    rancher-logging-kube-audit-fluentd-0             2/2     Running   0             36m
+    rancher-logging-rke2-journald-aggregator-c4kk4   1/1     Running   2 (49m ago)   59m
+    rancher-logging-rke2-journald-aggregator-rc4gf   1/1     Running   2 (29m ago)   59m
+    rancher-logging-rke2-journald-aggregator-sc56f   1/1     Running   2 (38m ago)   59m
+    rancher-logging-root-fluentbit-fh9v9             1/1     Running   2 (49m ago)   59m
+    rancher-logging-root-fluentbit-v65sz             1/1     Running   2 (38m ago)   59m
+    rancher-logging-root-fluentbit-xkp8p             1/1     Running   2 (29m ago)   59m
+    rancher-logging-root-fluentd-0                   2/2     Running   0             36m
+    ```  
+1. kube-audit files
+1. Check the kube-apiserver has param of audit
+    ```
+        n1-103:~ # ps aux | grep kube-apiserver
+        root      2819 41.7  2.6 3468424 2196848 ?     Ssl  12:25  56:05 kube-apiserver --audit-policy-file=/etc/rancher/rke2/config.yaml.d/92-harvester-kube-audit-policy.yaml --audit-log-path=/var/lib/rancher/rke2/server/logs/audit.log
+    ```
+1. The /var/lib/rancher/rke2/server/logs/audit.log have increasing content
```

Comment:
so in single node case, we don't need to check this?

Reply:
Thanks for the check, I remove the test result separate sections and combine single and multi node check point together